cmd/argocd/commands/utils/prompt.go

```diff
@@ -0,0 +1,30 @@
+package utils
+
+import (
+	"github.com/argoproj/argo-cd/v2/pkg/apiclient"
+	"github.com/argoproj/argo-cd/v2/util/cli"
+	"github.com/argoproj/argo-cd/v2/util/localconfig"
+)
+
+type Prompt struct {
+	enabled bool
+}
+
+func NewPrompt(clientOpts *apiclient.ClientOptions) (*Prompt, error) {
+	localCfg, err := localconfig.ReadLocalConfig(clientOpts.ConfigPath)
+	if err != nil {
+		return nil, err
+	}
+
+	return &Prompt{
+		enabled: localCfg.PromptsEnabled,
+	}, nil
+}
+
+func (p *Prompt) Confirm(message string) bool {
```

Comment:
Cover it with tests

Reply:
Covered the negative case where `p.enabled` is `false`